### PR TITLE
NEP-3417: PPS Quick Filter Labels

### PR DESCRIPTION
**Pull request by dn-readytech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,8 +128,10 @@ The following haml snippet is an example of the code to include on your html pag
         %dt.label{  :data => { value: "Rating" } }
     %dl.quick-filters
       %dl.author-block
+        %dt.label{ data: "Author Block Label"}
```
Comment (jbat) [inline README.md]: should this be the same configuration as the others? ie
{ :data => { value: "Author Block Label" }}

File: README.md [hunk 2]
```diff
@@ -128,8 +128,10 @@ The following haml snippet is an example of the code to include on your html pag
...
         %dl.mine-button
           %dt.label{ data: { value:  'Mine'} }
+          %dt.tooltip{ data: "My Tooltip"}
```
Comment (jbat) [inline README.md]: also here

File: src/components/QuickFilters/QuickFiltersBlock/QuickFiltersBlock.react.js [hunk 3]
```diff
@@ -4,20 +4,26 @@ export class QuickFiltersBlock extends React.Component {
...
+        return (
+          <QuickFiltersButton filters={filters[filter]} name={filter} blockName={this.state.name}/>
+        );
+      }
     }, this);
     return (
-      <div className="btn-group quick-filters-block">
-        {buttons}
+      <div>
+        {this.props.label}
```
Comment (jbat) [inline src/components/QuickFilters/QuickFiltersBlock/QuickFiltersBlock.react.js:23]: I am assuming we will need to style this label as well .. so might need a span and class

**Review by jbat — COMMENTED** [review]
Just a few questions

**Review by msxavi — COMMENTED** [review]
Seems good! Just one doubt regarding version control; I can't say for sure, but since we have been adding new features, the MINOR version is the one supposed t o be updated, right? (http://semver.org/). Shouldn't it be version "1.8.0"?

File: src/components/QuickFilters/QuickFiltersBlock/QuickFiltersBlock.react.js (hunk 3 above)
Comment (dn-readytech) [reply src/components/QuickFilters/QuickFiltersBlock/QuickFiltersBlock.react.js:23]: It's styled in the other PR.

**Review by jbat — APPROVED** [review]
+1

**Pull request merged**